starter-examples/starter-ranking.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
<p>Can we let the sdk handle computations with tru.compute_all instead of computing separately then adding? Relatedly, do we still need model output context? See the performance notebook for reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Unfortunately, we can't automatically compute predictions for ranking projects at the moment. This is an artifact of how we store predictions (i.e., with group/item IDs). For reference, this is the error message users will see if they try to automatically compute predictions via the SDK: <code>ValueError: Currently cannot ingest predictions automatically for ranking projects as they require group and item ID column names!</code>  </p><p> </p><p>I did remove the <code>model_output_context</code> calls, and I separated this cell into three separate cells to improve readability. </p>